reduce wakeups; this is a timeout that's only relevant if select()
were to be broken, which it isn't; and on windows the tool already has
a long timeout



--- gnupg-2.1.11/agent/gpg-agent.c~	2016-01-26 09:36:37.000000000 +0000
+++ gnupg-2.1.11/agent/gpg-agent.c	2016-04-25 03:00:45.549085574 +0000
@@ -279,7 +279,7 @@
 # define TIMERTICK_INTERVAL          (4)
 # define CHECK_OWN_SOCKET_INTERVAL  (60)
 #else
-# define TIMERTICK_INTERVAL          (2)
+# define TIMERTICK_INTERVAL         (30)
 # define CHECK_OWN_SOCKET_INTERVAL  (60)
 #endif
 
